src/styles/blocks/calendar.scss

```
@@ -0,0 +1,46 @@
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-flow: row wrap;
+  font-family: $font-family;
+  font-size: $font-size;
+  gap: $grid-gap;
+  padding-inline: $calendar-inline-padding;
+  width: $calendar-width;
+
+  &__day {
+    align-items: center;
+    background-color: $day-color;
+    border: $day-border;
+    box-sizing: border-box;
+    cursor: pointer;
+    display: flex;
+    height: $day-size;
+    justify-content: center;
+    transition: all $transition-duration;
```

don't use it like that, because it's the same as transition: $transition-duration
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition: color $transition-duration, transform $transition-duration;
```